test/JDBC/expected/BABEL-3646-vu-verify.out

```diff
@@ -0,0 +1,826 @@
+SELECT * FROM BABEL_3646_vu_prepare_v1
+GO
+~~START~~
+date
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: cannot cast type xml to date.)~~
+
+DROP VIEW BABEL_3646_vu_prepare_v1
+GO
+
+EXEC BABEL_3646_vu_prepare_p1
+GO
+~~START~~
+date
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: cannot cast type xml to date.)~~
+
+DROP PROCEDURE BABEL_3646_vu_prepare_p1
+GO
+
+SELECT BABEL_3646_vu_prepare_f1()
+GO
+~~START~~
+date
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: cannot cast type xml to date.)~~
+
+DROP FUNCTION BABEL_3646_vu_prepare_f1()
+GO
+
+SELECT * FROM BABEL_3646_vu_prepare_v2
```

Comment:
Is this a correct output?

Reply:
Its correct output for how Babelfish currently works. We allow a XML -> text and text -> date so this conversion works. In TSQL this will throw an error